python/test/pyRoot_TEST.py

```diff
@@ -0,0 +1,262 @@
+# Copyright (C) 2022 Open Source Robotics Foundation
+
+# Licensed under the Apache License, version 2.0 (the "License")
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#       http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import copy
+from ignition.math import Vector3d, Pose3d
+from sdformat import Error, Root, SDF_VERSION, SDF_PROTOCOL_VERSION, World
+import unittest
+
+
+class RootTEST(unittest.TestCase):
+
+  def test_default_construction(self):
+    root = Root()
+    # self.assertEqual(SDF_VERSION, root.version())
```

Comment:
can this expectation be enabled?

Reply:
enabled in https://github.com/ignitionrobotics/sdformat/pull/967/commits/1f0c329751801eda7c45911dc2fa73d87767dfc2